Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/Services/ComponentsController.cs

```diff
@@ -104,17 +106,27 @@ public HttpResponseMessage GetSuggestionRoles(string keyword, int roleGroupId, i
                     return this.Request.CreateResponse(HttpStatusCode.OK, new List<SuggestionDto>());
                 }
 
-                var matchedRoles = RoleController.Instance.GetRoles(this.PortalId)
-                    .Where(r => (roleGroupId == -2 || r.RoleGroupID == roleGroupId)
-                                && r.RoleName.IndexOf(keyword, StringComparison.InvariantCultureIgnoreCase) > -1
-                                   && r.Status == RoleStatus.Approved)
-                    .Select(r => new SuggestionDto()
+                var portalRoles = RoleController.Instance.GetRoles(this.PortalId);
+                
+
+                var matchedRoles = portalRoles.Where(r => (roleGroupId == -2 || r.RoleGroupID == roleGroupId)
+                                                          && r.RoleName.IndexOf(keyword, StringComparison.InvariantCultureIgnoreCase) > Null.NullInteger
+                                                          && r.SecurityMode != SecurityMode.SocialGroup
```

Comment:
The change to remove social groups seems separate from the intention of this PR.  Does this relate to adding Unauthenticated Users?  If not, let's tackle it as a separate issue and PR.  Thanks!